Add Cohere RAG tutorial link (#6442)
opensearch-project/documentation-website: _search-plugins/conversational-search.md

```
{% include copy-curl.html %}

## RAG pipeline

RAG is a technique that retrieves documents from an index, passes them through a seq2seq model, such as an LLM, and then supplements the static LLM information with the dynamically retrieved data in context.

As of OpenSearch 2.12, the RAG technique has only been tested with OpenAI models, the Anthropic Claude model on Amazon Bedrock, and Cohere Command models. 
{: .warning}

> Configuring the Cohere Command model to enable RAG requires using a post-processing function to transform the model output. For more details, you can refer to the [Cohere RAG Tutorial](https://github.com/opensearch-project/ml-commons/blob/2.x/docs/tutorials/conversational_search/conversational_search_with_Cohere_Command.md).

Reviewer: ```suggestion
Configuring the Cohere Command model to enable RAG requires using a post-processing function to transform the model output. For more details, see the [Cohere RAG Tutorial](https://github.com/opensearch-project/ml-commons/blob/2.x/docs/tutorials/conversational_search/conversational_search_with_Cohere_Command.md).
```

RAG is a technique that retrieves documents from an index, passes them through a seq2seq model, such as an LLM, and then supplements the static LLM information with the dynamically retrieved data in context.

> As of OpenSearch 2.11, the RAG technique has only been tested with OpenAI models, the Anthropic Claude model on Amazon Bedrock, and Cohere command models. 

Author: Updated this

{: .warning}

> Setting up Cohere's command model to enable RAG with chat requires modified request bodies than shown below, please refer to the [Cohere RAG Tutorial](https://github.com/opensearch-project/ml-commons/blob/2.x/docs/tutorials/conversational_search/conversational_search_with_Cohere_Command.md) for more details.

Reviewer: I'm planning to migrate the tutorials that live in ML Commons to the documentation. For now, we can link to GitHub, but this link will change once the migration is complete.
```suggestion
Configuring the Cohere Command model to enable RAG requires using a post-processing function to transform the model output. For more information, see [Cohere RAG Tutorial](https://github.com/opensearch-project/ml-commons/blob/2.x/docs/tutorials/conversational_search/conversational_search_with_Cohere_Command.md).
```

Author: SGTM - Updated the sentence
